### PR TITLE
Create pythonpackage.yml to run a CI build

### PR DESCRIPTION
**Pull request by LouisBrunner**

### CODE REVIEW

File: .github/workflows/publish.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+name: Publish on release
```
Comment (Charlesworth) [inline .github/workflows/publish.yml]: Is this not a bit pointless? Can we get rid of this file and add it later if we need it, as it's redundant?

File: evasdk/__init__.py [hunk 2]
```diff
@@ -1,3 +1,72 @@
+"""Eva Python SDK
```
Comment (Charlesworth) [inline evasdk/__init__.py:1]: Not sure about copying the README here, DRY and all. perhaps just link to the README or the github web page?

**Review by Charlesworth — CHANGES_REQUESTED** [review]
Looks great, linting and removal of unused deps, nice. Pipelines work well. Couple of comments, Also, I'll test it on a Robot just to make sure the linting didn't mess anything up

File: evasdk/__init__.py (hunk 2 above)
Comment (LouisBrunner) [reply evasdk/__init__.py:1]: That's standard Python practice, the same way you add documentation on every class.

File: .github/workflows/publish.yml (hunk 1 above)
Comment (LouisBrunner) [reply .github/workflows/publish.yml]: We can, but I think we should automate publishing, no need to have it as a manual process when it's automated in 10 lines and anybody with the right permission can trigger it.

Comment (Charlesworth) [reply .github/workflows/publish.yml]: Totally agree, but this doesn't do that yet and probably could be another PR in itself.

Comment (LouisBrunner) [reply .github/workflows/publish.yml]: Oh sorry, I forgot I didn't finish to write it

Comment (LouisBrunner) [reply .github/workflows/publish.yml]: This should now be fixed

Comment (LouisBrunner) [reply .github/workflows/publish.yml]: This is now gone and will be in another PR

**Review by Charlesworth — APPROVED** [review]

**Pull request merged**